client/post-editor/editor-drawer/index.jsx

```diff
@@ -240,7 +242,23 @@ const EditorDrawer = React.createClass( {
 					isNew={ this.props.isNew }
 				/>
 			</AccordionSection>
-		)
+		);
+	},
+
+	renderSeo: function() {
+		if ( ! config.isEnabled( 'manage/advanced-seo' ) || ! this.props.site ) {
```

Comment:
I wonder if there's a graceful way to show some kind of "Checking for business plan" message and fade away the according for free plans automatically so that people won't always have it showing up in an otherwise dense config area?